exchange/events_test.go

```diff
@@ -13,6 +13,7 @@ func Test_eventsData_makeBidExtEvents(t *testing.T) {
 		enabledForAccount bool
 		enabledForRequest bool
 		bidType           openrtb_ext.BidType
+		generatedBidId    string
```

Comment:
**Nitpick**: Can you please update this to `generatedBidID` as well for consistency?

Reply:
Yes, done!